### ISSUE TITLE
Creating form builder with two/three column layout

### ISSUE THREAD

**Issue by manubenjamin** [kelp404/angular-form-builder#89]
I need to create form builder with two or three column layout (Drag and drop component to column and reposition it within the columns). I have tried the same by adding different fb-builder in each column. But for me, I am not able to drag and drop the component from one column to other.

Can you help me to fix the issue.

**Comment by huei90** [comment]
Are you able to dnd on the demo page?

**Comment by manubenjamin** [comment]
Yes, but I am not dragging and dropping component from fb-component. For me there is some dynamic list and based on the input, I am creating the component and adding to fb-builder. After adding to fb-builder i am able to drag and reposition the component inside the fb-builder. But moving component from one fb-builder to other is not working.

**Comment by kimthuy** [comment]
I recommend to create multiple fb-builder

**Comment by santosh-k1** [comment]
Hi, You need to edit in angular-form-builder.js steps are as below -

- i have added col-md-3 which will help me to create 4 columns
 template: "<div class='form-horizontal'>\n    <div class='fb-form-object-editable col-md-3' ng-repeat=\"object in formObjects\"\n        fb-form-object-editable=\"object\"></div>\n</div>",

- Blank area where you have drag by default it shows 100 % but you need to use col-md-3 so that it shows same area where you want to place the controls 
$empty = $("<div class='fb-form-object-editable empty col-md-3'></div>");
I hope it will help you to fix the column issue.

**Comment by Hirenp75** [comment]
Hi santosh-k1,

i have added col-md-3 for create the 4 columns and col-md-4 for create the 3 columns

my template : "<div class=\"form-horizontal\">\n <div class=\"row\"> <div class=\"fb-form-object-editable col-md-3\" ng-repeat=\"object in formObjects\" fb-form-object-editable=\"object\"> </div>\n </div>\n </div>"

but still drag and drop functionality isn't working for me.so can you help for me solved 4 column and column related issue.

If you have any other solution please let me know.it's great for me.

Thanks

**Comment by devinternago** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
092000, @paulo9mv, @immint023, @x44x45x41x4E, @JohannesKlauss, @wdkang123, @LimGigap, @legend91325, @KtfwyCJ, @ilittlesun, @MariamFahmy98, @lonkelle, @LiveFly, @chuckcarpenter, @mzabriskie, @Achyutem, @thesanjeevsharma, @vintagesucks, @Evild67, @Domon
